README.md

```diff
@@ -7,15 +7,52 @@
+```sh
+  "release": "local-latest",
+  "images": [
+      {
+          "name": "controller",
+          "path": "ixiacom/ixia-c-controller",
+          "tag": "0.0.1-2994",
+          "args": ["--accept-eula"]
+      },
+      {
+          "name": "gnmi-server",
+          "path": "ixiacom/ixia-c-gnmi-server",
+          "tag": "1.8.3",
+          "command": ["python3", "-m", "otg_gnmi"]
+      },
+      {
+          "name": "traffic-engine",
+          "path": "docker-local-ixvm-lbj.artifactorylbj.it.keysight.com/athena-traffic-engine",
+          "tag": "1.4.1.29",
+          "env": {
+              "OPT_LISTEN_PORT": "5556"
+          }
+      },
+      {
+```
+
+Following customizations are supported for the image components; as shown in the sample configmap above:
+1. Specify a custom location ("path").
+2. Specify a custom version ("tag").
+3. Specify custom "command", "args" or "env" to override standard built-in ones.
+
+If some of the required images are not publicly available and distributed under license, user would be required to create a secret with docker authorization details as supplied by Keysight. The operator would refer to this secret for deploying from private repository.<br/><br/>
 
-<img src="Ixia_Operator.jpg">
+<kbd> <img src="Ixia_Operator.jpg"> </kbd><br/><br/>
 
-The operator deploys one single Controller pod with Ixia-c, gNMI and gRPC containers for user control, management and stats report of Ixia network devices; and Ixia network devices for control and data plane. The deployed Ixia resource release versions are anchored and dictated by the Ixia-c release as defined in the KNE config file.
+The operator deploys one single Controller pod with Ixia-c, gNMI and gRPC containers for user control, management and statistics reporting of Ixia network devices; and Ixia network devices for control and data plane. The deployed Ixia resource release versions are anchored and dictated by the Ixia-c release as defined in the KNE config file.
 
 ## Deployment
 
```

Comment:
Add a section explaining our CRD for people who would want to use it outside of KNE.

Reply:
Done.